framework/src/modules/poa/commands/register_authority.ts

```diff
@@ -50,20 +49,12 @@ export class RegisterAuthorityCommand extends BaseCommand {
 		context: CommandVerifyContext<RegisterAuthorityParams>,
 	): Promise<VerificationResult> {
 		const { name } = context.params;
-		try {
-			validator.validate(registerAuthoritySchema, context.params);
-		} catch (err) {
-			return {
-				status: VerifyStatus.FAIL,
-				error: err as Error,
-			};
-		}
 
 		if (!POA_VALIDATOR_NAME_REGEX.test(name)) {
 			throw new Error(`Name does not comply with format ${POA_VALIDATOR_NAME_REGEX.toString()}.`);
 		}
 
-		const nameExists = await this.stores.get(NameStore).has(context, Buffer.from(name));
+		const nameExists = await this.stores.get(NameStore).has(context, Buffer.from(name, 'utf-8'));
```

Comment:
I think by default its utf-8 so we don't have to mention it

Reply:
The audit report pointed out that in some places we use `Buffer.from` and some places `Buffer.from(..,'utf8') and that should be consistent which is why I updated it here.